### PR TITLE
fix(spinner): added svg variation

### LINKED ISSUE #2257: Spinner artifacts in Firefox
I'm seeing this in Firefox

Screen Shot 2019-09-12 at 10 44 39 AM

### LINKED ISSUE #3640: Spinner: time travel paradox
_Great Scott! Strange things are afoot at the Circle, K?_

Spinners spin clockwise as they're abstract representations of the forward motion of the hands of a clock. 

PatternFly's spinner moves counterclockwise (or [anticlockwise in the UK](https://wikidiff.com/counterclockwise/anticlockwise)), indicating that waiting on something to finish will finally happen _in the past_ somehow. :wink:

Additionally, mixing and matching spinners, as we're doing in Cockpit now (with the goal of phasing out PatternFly 3 completely, eventually) can make a person dizzy.

Solution? Reverse the polarity of the spin rotation.

### PR DESCRIPTION
**Pull request by mattnolting**
closes #2257 
closes #3640 

This PR converts the spinner to an SVG. It retains the same base variables.

Preview: https://patternfly-pr-3690.surge.sh/components/spinner/

### CODE REVIEW

**Review by mcarrano — APPROVED** [review]
Looks good to me. Thanks @mattnolting !

**Review by christiemolloy — APPROVED** [review]
Nice!!

File: src/patternfly/components/Spinner/spinner.scss [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
-  // Modifier variables for main spinner
-  // Width and Height must be the same
+  // sizes
   --pf-c-spinner--m-sm--diameter: var(--pf-global--icon--FontSize--sm);
   --pf-c-spinner--m-md--diameter: var(--pf-global--icon--FontSize--md);
   --pf-c-spinner--m-lg--diameter: var(--pf-global--icon--FontSize--lg);
   --pf-c-spinner--m-xl--diameter: var(--pf-global--icon--FontSize--xl);
 
+  width: var(--pf-c-spinner--Width);
+  height: var(--pf-c-spinner--Height);
```
Comment (mcoker) [inline src/patternfly/components/Spinner/spinner.scss:16]: You also have width/height declared under `span.pf-c-spinner` set to the same values - do you need them both?

Comment (mattnolting) [reply src/patternfly/components/Spinner/spinner.scss:16]: Nope! Updated.

**Review by mcoker — APPROVED** [review]
L✅TM!

**Pull request merged**